### ISSUE TITLE
Could not pass event ResidenceChangedEvent to Residence v4.9.3.3

### ISSUE THREAD

**Issue by Baterka** [Zrips/Residence#698]
Sometimes this prints out, but I am not sure when...

```
[21:42:26] [Server thread/ERROR]: Could not pass event ResidenceChangedEvent to Residence v4.9.3.3
[21:42:26] [Server thread/ERROR]: Suspected Plugins: 
[21:42:26] [Server thread/ERROR]: 	Residence{enabled,ver=4.9.3.3,path=plugins/Residence4.9.3.3.jar}
[21:42:26] [Server thread/ERROR]: Exception details below: 
java.lang.NullPointerException: null
	at com.bekvon.bukkit.residence.listeners.ResidencePlayerListener1_9.onResidenceChange(ResidencePlayerListener1_9.java:68) ~[?:?]
	at com.destroystokyo.paper.event.executor.asm.generated.GeneratedEventExecutor916.execute(Unknown Source) ~[?:?]
	at org.bukkit.plugin.EventExecutor.lambda$create$1(EventExecutor.java:69) ~[1.16.5-patched.jar:git-Yatopia-ver/1.16.5-33]
	at co.aikar.timings.TimedEventExecutor.execute(TimedEventExecutor.java:80) ~[1.16.5-patched.jar:git-Yatopia-ver/1.16.5-33]
	at org.bukkit.plugin.RegisteredListener.callEvent(RegisteredListener.java:70) ~[1.16.5-patched.jar:git-Yatopia-ver/1.16.5-33]
	at org.bukkit.plugin.SimplePluginManager.callEvent(SimplePluginManager.java:611) ~[1.16.5-patched.jar:git-Yatopia-ver/1.16.5-33]
	at com.bekvon.bukkit.residence.listeners.ResidencePlayerListener.handleNewLocation(ResidencePlayerListener.java:2166) ~[?:?]
	at com.bekvon.bukkit.residence.listeners.ResidencePlayerListener.onPlayerMove(ResidencePlayerListener.java:2002) ~[?:?]
	at com.destroystokyo.paper.event.executor.asm.generated.GeneratedEventExecutor947.execute(Unknown Source) ~[?:?]
	at org.bukkit.plugin.EventExecutor.lambda$create$1(EventExecutor.java:69) ~[1.16.5-patched.jar:git-Yatopia-ver/1.16.5-33]
	at co.aikar.timings.TimedEventExecutor.execute(TimedEventExecutor.java:80) ~[1.16.5-patched.jar:git-Yatopia-ver/1.16.5-33]
	at org.bukkit.plugin.RegisteredListener.callEvent(RegisteredListener.java:70) ~[1.16.5-patched.jar:git-Yatopia-ver/1.16.5-33]
	at org.bukkit.plugin.SimplePluginManager.callEvent(SimplePluginManager.java:611) ~[1.16.5-patched.jar:git-Yatopia-ver/1.16.5-33]
	at net.minecraft.server.v1_16_R3.PlayerConnection.a(PlayerConnection.java:1351) ~[1.16.5-patched.jar:git-Yatopia-ver/1.16.5-33]
	at net.minecraft.server.v1_16_R3.PacketPlayInFlying.a(SourceFile:122) ~[1.16.5-patched.jar:git-Yatopia-ver/1.16.5-33]
	at net.minecraft.server.v1_16_R3.PacketPlayInFlying$PacketPlayInPosition.a(SourceFile:56) ~[1.16.5-patched.jar:git-Yatopia-ver/1.16.5-33]
	at net.minecraft.server.v1_16_R3.PlayerConnectionUtils.lambda$ensureMainThread$1(PlayerConnectionUtils.java:43) ~[1.16.5-patched.jar:git-Yatopia-ver/1.16.5-33]
	at net.minecraft.server.v1_16_R3.TickTask.run(SourceFile:18) ~[1.16.5-patched.jar:git-Yatopia-ver/1.16.5-33]
	at net.minecraft.server.v1_16_R3.IAsyncTaskHandler.executeTask(IAsyncTaskHandler.java:136) ~[1.16.5-patched.jar:git-Yatopia-ver/1.16.5-33]
	at net.minecraft.server.v1_16_R3.IAsyncTaskHandlerReentrant.executeTask(SourceFile:23) ~[1.16.5-patched.jar:git-Yatopia-ver/1.16.5-33]
	at net.minecraft.server.v1_16_R3.IAsyncTaskHandler.executeNext(IAsyncTaskHandler.java:109) ~[1.16.5-patched.jar:git-Yatopia-ver/1.16.5-33]
	at net.minecraft.server.v1_16_R3.MinecraftServer.bb(MinecraftServer.java:1218) ~[1.16.5-patched.jar:git-Yatopia-ver/1.16.5-33]
	at net.minecraft.server.v1_16_R3.MinecraftServer.executeNext(MinecraftServer.java:1211) ~[1.16.5-patched.jar:git-Yatopia-ver/1.16.5-33]
	at net.minecraft.server.v1_16_R3.IAsyncTaskHandler.awaitTasks(IAsyncTaskHandler.java:119) ~[1.16.5-patched.jar:git-Yatopia-ver/1.16.5-33]
	at net.minecraft.server.v1_16_R3.MinecraftServer.sleepForTick(MinecraftServer.java:1187) ~[1.16.5-patched.jar:git-Yatopia-ver/1.16.5-33]
	at net.minecraft.server.v1_16_R3.MinecraftServer.w(MinecraftServer.java:1027) ~[1.16.5-patched.jar:git-Yatopia-ver/1.16.5-33]
	at net.minecraft.server.v1_16_R3.MinecraftServer.lambda$a$0(MinecraftServer.java:176) ~[1.16.5-patched.jar:git-Yatopia-ver/1.16.5-33]
	at java.lang.Thread.run(Thread.java:834) [?:?]
```

**Comment by janvrska** [comment]
#693

**Comment by lalalalajciyiy** [comment]
Have the same problem... Still not fixed?

**Comment by StraikerinaCZ** [comment]
#693

**Comment by lalalalajciyiy** [comment]
Pleaseeeeeeeeeeeeeeeee..... fix it ...... Its only spamming my console...

**Comment by Zrips** [comment]
This was fixed in v4.9.3.4 version
